src/Propulsion/Streams.fs

```diff
@@ -342,7 +342,7 @@ module Scheduling =
                         malformed <- malformed + 1
                         malformedB <- malformedB + sz
                         malformedE <- malformedE + state.EventsCount
-                    elif state.HasGap then
+                    elif not state.IsComplete then
```

Comment:
🤔 I think it reads better before. I know what a gap is in this context, but completeness could be anything

Reply:
Maaan. And there was me thinking "You Complete Me. how nice"
QueuedIsAtWritePos is my next offer...

Reply:
next commit probably expresses things better (and also has some de morgan things to double check)